plugin/core/panels.py

```diff
@@ -37,9 +37,19 @@ class PanelName:
 
 @contextmanager
 def mutable(view: sublime.View) -> Generator:
+    # Unset the read-only flag temporarily
     view.set_read_only(False)
-    yield
-    view.set_read_only(True)
+    try:
+        yield
+    finally:
```

Comment:
Why within an exception handler now? Do you of any case when this could happen?

Reply:
yes but I'm not sure why would we get exceptions triggered within "with" and whether we want to hide them like that

Reply:
the exception is not consumed, only the `finally` block is executed to do cleanup and the exception will then keep going up the call stack.